azure-communication-ui/build.gradle

```diff
@@ -46,7 +46,7 @@ buildscript {
             // and enable the beta features
 
             // Latest GA version of the SDK
-            azure_calling_sdk_version_ga = '2.12.0'
+            azure_calling_sdk_version_ga = '2.13.0-beta.1'
 
             // Newest version of the SDK, can be beta
             azure_calling_sdk_version_beta = '2.12.0-beta.1'
```

Comment:
I know there are no flags, but once this is merged, we are hard-committed to 2.13 GA (which might be delayed due to TDL bugs).

Flags, and using the _beta field would help in mitigating this risk, but it's a lot to ask at this point. 

If something comes up and we need adjust path to GA without RTT (for any reason), we'll have to strategically flag to disable when going back to 2.12, or hotfix on top of the last release. It's worth acknowledging this risk before merging. Best case it's a non-issue.

Reply:
We did discuss it: as there is no other feature in development that may go sooner, I did not add a feature flag. Otherwise, it would be much harder to do all those refactorings and renamings.

Reply:
We can just do it surgically if it comes down to it, it'd be way less work than doing the entire feature. I'm not expecting on this PR, just bringing up the risk.